config/constants.yml

```diff
@@ -0,0 +1,15 @@
+default: &default
```

Comment:
I'm wondering what your reasoning is behind using a yml file instead of putting constants in a model.

Normall, application configurations like this aren't tracked by git, such as the `sftp.yml` file in umaps-rails that contains credentials to log into sftp servers, or the `database.yml` file which differs across whatever server its on. If a sysadmin sees this file on the server, they might think that they could just edit the file on the server, because that's the usual case.
So that's a good reason against doing constants this way--I'm wondering what your reason _for_ it is. If it's just that that's the only way you know how to do constants, please consider `TPP::Purchase::CONSTANTS` which I named `constants` because I didn't have anything better in mind right then but in retrospect was a terrible decision.

Reply:
Partly this was me copying the way I remembers umaps handling constants. Partially the reasoning was that a .yml would be easier for a non programmer to adjust if these parameters needed to be adjusted. Also because this project is open source, it makes sense to abstract out constants like this a configuration files so that if someone else were to use this app, though unlikely, they would have an easier time setting it up and wouldn't need to go and adjust models.